.travis.yml

```diff
@@ -0,0 +1,19 @@
+sudo: required
+
+language: minimal
+
+services: docker
+
+install:
+    - ln -s .env.dist .env
+    - travis_retry docker-compose pull
+
+before_script:
+    - docker-compose up -d
+
+script:
+    - .travis/smoke
```

Comment:
Might be worth standardising this name like https://github.com/libero/dummy-api/pull/10/files#diff-354f30a63fb0907d4ad57269548329e3R17.